ansible/roles/nginx-prometheus/templates/nginx.conf

```diff
@@ -8,7 +8,17 @@ events {
   worker_connections 448;
 }
 
+lock_file /run/ngxprom.lock;
```

Comment:
I guess it was not obvious to me since I was just reading the diff and did not run `git grep`.